package.json

```diff
@@ -18,7 +18,7 @@
   },
   "main": "./client/out/client.js",
   "scripts": {
-    "package": "vsce package --yarn",
+    "package": "vsce package",
```

Comment:
The dev readme doesn't use yarn, so I fixed this script. Should maybe be in a separate commit, but it's a pretty small fix.